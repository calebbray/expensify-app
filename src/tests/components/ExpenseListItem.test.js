import React from 'react';
import { shallow } from 'enzyme';
import ListItem from '../../components/ExpensesListItem';
import expenses from '../fixtures/expenses';
import toJSON from 'enzyme-to-json';

test('Should render list item', () => {
  const wrapper = shallow(<ListItem {...expenses[0]} />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});