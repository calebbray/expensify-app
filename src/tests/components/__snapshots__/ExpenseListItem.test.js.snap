// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render list item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      gum
    </h3>
  </Link>
  <p>
    1950
     - 
    0
  </p>
</div>
`;
